scrapbook-tutorial/public/index.html

```diff
@@ -28,7 +28,7 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    <div id="root" class="root"></div>
```

Comment:
You probably don't need to add the class here.  If you want to style the root you can change the css selector to #root instead of .root without needing to modify the html.

Reply:
Done